src/software/simulated_tests/terminating_validation_functions/robot_in_polygon_validation.h

```diff
@@ -5,11 +5,10 @@
 
 /**
  * Checks if the robot represented by robot_id is in the polygon
```

Comment:
please update this comment
```suggestion
 * Checks if the some given number of robots are in the polygon
```